css/parser/tests/recovery/delim-token/asterisk/output.swc-stderr

```
@@ -1,6 +0,0 @@
-error: Expected Declaration value
```

Same here, it should report an error.